File: 20161110-java-racing_car_katas_3/src/test/java/tddmicroexercises/turnticketdispenser/TicketDispenserTest.java
package tddmicroexercises.turnticketdispenser;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.CoreMatchers.notNullValue;
import static org.junit.Assert.assertThat;

public class TicketDispenserTest {

    @Test
    public void getTurnTicket_creates_a_new_ticket_with_turn_number_to_ticket() {
        TicketDispenser dispenser = new TicketDispenser();

        TurnTicket ticket = dispenser.getTurnTicket();

        assertThat(ticket, notNullValue());
        assertThat(ticket.getTurnNumber(), is(0));
    }

}

File: 20170530-csharp-ticket-dispenser/TurnTicketDispenser.Tests/TicketDispenserTest.cs
using NUnit.Framework;
namespace TDDMicroExercises.TurnTicketDispenser.Tests
{
    [TestFixture]
    public class TicketDispenserTest
    {
        [Test]
        public void GetTurnTicket_should_return_a_new_ticket_using_unique_sequence_number() {
            TicketDispenser ticketDispenser = new TicketDispenser();

            TurnTicket ticket = ticketDispenser.GetTurnTicket();

            Assert.AreEqual(0, ticket.TurnNumber);
        }
    }
}
